test/new_relic/agent/instrumentation/rails/custom_events_subscriber_test.rb

```diff
@@ -0,0 +1,95 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative '../../../../test_helper'
+require 'new_relic/agent/instrumentation/custom_events_subscriber'
+
+module NewRelic::Agent::Instrumentation
+  class CustomEventsSubscriberTest < Minitest::Test
+    TOPIC = 'abre los ojos'
```

Comment:
Does ActiveSupport allow topics with whitespace?

Reply:
Yep!

```ruby
#!/usr/bin/env ruby
# frozen_string_literal: true

require 'bundler/inline'

gemfile do
  source 'https://rubygems.org'

  gem 'rails', require: false
end

require 'active_support/notifications'
require 'active_support/isolated_execution_state'

EVENT_NAME = 'abre los ojos'

ActiveSupport::Notifications.subscribe EVENT_NAME do |name, started, finished, unique_id, data|
  puts "name: >>#{name}<<"
  puts "started: >>#{started}<<"
  puts "finished: >>#{finished}<<"
  puts "unique_id: >>#{unique_id}<<"
  puts "data: >>#{data.inspect}<<"
end

ActiveSupport::Notifications.instrument EVENT_NAME, this: :data do
  rand(1148)
end
```

Reply:
Awesome, thank you!